src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -560,50 +577,37 @@ private void processLogsAndUpload() throws InterruptedException {
             }
             AtomicReference<Optional<ComponentLogFileInformation>> componentLogFileInformation =
                     new AtomicReference<>(Optional.empty());
+            AtomicBoolean isActiveFileCompleted = new AtomicBoolean(false);
```

Comment:
This does not necessarily to be a AtomicBoolean, this type only helps changing value in the temporary `ACTIVE_FILE_FEATURE_ENABLED_FLAG`.